### PR TITLE
Update Path filter checks

### PR DESCRIPTION
**Pull request by vitalikda**
# Description

This aims to improve the reliability and consistency of change detection for the CLI and Packages 

# Type of change

- [x] 🐛 Fix

### CODE REVIEW

File: .github/workflows/cli.yml [hunk 1]
```diff
@@ -40,8 +40,7 @@ jobs:
           filters: |
             build:
               - 'cli/**/*'
-              - '.github/**/*'
-              - '!.github/workflows/packages.yml'
+              - '.github/workflows/cli.yml'
```
Comment (vitalikda) [inline .github/workflows/cli.yml:43]: I updated the filter due to it affects all files except `packages.yml`
Do we need to run CLI workflow for anything else?

Comment (obmarg) [reply .github/workflows/cli.yml:43]: `cli.yml` definitely pulls in a bunch of other files inside `.github/workflows` & `.github/actions`.  We probably want to list them here as well?  or add some more ignores for whatever prompted this PR in the first place?

Comment (vitalikda) [reply .github/workflows/cli.yml:43]: @obmarg wanted to be explicit here 

Looking into cli related workflows 
- `.github/workflows/cli.yml` - included in filter 
- `.github/workflows/cli-partial-release.yml` - triggered by `cli.yml`
- `.github/workflows/sanitize-alert-schedule.yml` - triggered by cron or manually 

So I do not think workflows besides cli should be included. Actions on the other hand should - I will add them

Wdyt?

Comment (obmarg) [reply .github/workflows/cli.yml:43]: Yeah, we can try that.

Comment (vitalikda) [reply .github/workflows/cli.yml:43]: Added `.github/actions` 👍

**Pull request merged**